packages/@tinacms/mdx/src/stringify/marks.ts

```diff
@@ -77,7 +77,7 @@ const replaceLinksWithTextNodes = (content: Plate.InlineElement[]) => {
                 type: 'link',
                 url: item.url,
                 title: item.title,
-                children: [text({ text: a.value })],
+                children: [a],
```

Comment:
Nothing failed when I removed this. Its inclusion was stripping the `code: true` property required to properly linkify the `inlineCode` element.

Reply:
Thanks @mAAdhaTTah, `inlineCode` and `text` nodes happen to have the [same shape](https://github.com/syntax-tree/mdast#inlinecode), and they're the only types that should get through to this point, so your fix makes sense. 👍